format/index.md

```diff
@@ -189,10 +189,12 @@ be represented under the "links object".
 
 #### Complex Attributes <a href="#document-structure-resource-object-complex-attributes" id="document-structure-resource-object-complex-attributes"></a>
 
-"[Complex attributes]" are [attributes] whose value is an object or array with
-any level of nesting. An object that constitutes or is contained in a complex
-attribute **MUST** reserve the `id`, `type`, `links`, and `meta` members for future
-use.
+"[Complex attributes]" are [attributes] whose value is an object or array
+([example](/examples/#complex-attributes)).
+
+JSON API permits embedding arbitrary data structures as complex attributes in
+resource objects. However, any object in a complex attribute **MUST** reserve
```

Comment:
Sorry I'm just responding to this! I've been bogged down with stuff :)

I think your simplifications were really good overall. It was only the removal of the "constitutes or is contained in" phrase that tripped me up initially. I agree, though, that it's unlikely that "any object in a complex attribute's value" could reasonably be misinterpreted. I still think there might be a value in precision here, but I guess I don't have strong feelings either way.